src/coreclr/debug/runtimeinfo/contracts.jsonc

```diff
@@ -10,6 +10,6 @@
 // so to conditionally include contracts, put additional contracts in a separate file
 {
   "Exception": 1,
-  "Thread": 1,
-  "SOSBreakingChangeVersion": 1 // example contract: "runtime exports an SOS breaking change version global"
+  "Loader": 1,
+  "Thread": 1
```

Comment:
Not sure if you meant to delete "SOSBreakingChangeVersion".  (I think the comment is not really true anymore - it's more than an example - we do depend on that contract in cdacreader)